types/ember__routing/tsconfig.json

```diff
@@ -3,7 +3,7 @@
         "module": "commonjs",
         "target": "es5",
         "lib": [
-            "es6",
+            "es2018",
```

Comment:
`Promise.prototype.finally` is only available with `es2018`, I'm not sure what else this impacts though 💭